egs2/mls/asr2/conf/slurm.conf

```diff
@@ -0,0 +1,18 @@
+# Default configuration
```

Comment:
please set it as default (our CI check would ask for that

Reply:
Applied in https://github.com/espnet/espnet/pull/5441/commits/91c2b06069b95492b9f6eee9ffb5e1cc84bdb896 Thanks 👍